### ISSUE TITLE
Need to update "Prerequisites" section documentation

### ISSUE THREAD

**Issue by nicosocoro** [vladkol/gstreamer-netcore#5]
### Considerations

- I'm using [GStreamer mingw version 1.18.2](https://gstreamer.freedesktop.org/data/pkg/windows/1.18.2/mingw/). Not [version 1.16.2](https://gstreamer.freedesktop.org/data/pkg/windows/1.16.2/) as indicated in the documentation. 

### Issue description
In [README's Prerequisites section (working on Windows)](https://github.com/vladkol/gstreamer-netcore#if-you-develop-on-windows-please-read-this), it states we need to add two new values to PATH environment variable (EV):

- %GSTREAMER_1_0_ROOT_X86_64%\bin
- %GSTREAMER_1_0_ROOT_X86_64%\lib\gstreamer-1.0

However, when we install the [GStreamer mingw version](https://gstreamer.freedesktop.org/data/pkg/windows/1.18.2/mingw/), it creates the **GSTREAMER_1_0_ROOT_MINGW_X86_64** EV (_notice the extra_ MINGW _between_ ROOT _and_ X86_64).

At first, I got an error related to not being able to find **gstreamer-1.0-0.dll** (similar as reported in [Issue#1: You are missing gstreamer-1.0-0.dll](https://github.com/vladkol/gstreamer-netcore/issues/1)). Although, adding MINGW made it work out.

To sum up, the two above values should be, respectively, updated to:

- %GSTREAMER_1_0_ROOT_MINGW_X86_64%\bin
- %GSTREAMER_1_0_ROOT_MINGW_X86_64%\lib\gstreamer-1.0

**Comment by MCREAPER-dotcom** [comment]
As i understand MINGW can avoided if you just use:
GLib.GType.Register(Gst.Element.GType, typeof(Gst.Element));
GLib.GType.Register(Gst.Pipeline.GType, typeof(Gst.Pipeline));
when innit you progect.

I won't say for sure, but it helped me compile on MSVC
